ProjectTemplates/ArcGISRuntime.AR/Template.Forms/Android/Android.csproj

```diff
@@ -53,9 +53,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991211" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.0" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.1874" />
```

Comment:
Is it worth going straight to latest stable?

Reply:
It matches our requirements. So no. Users are free to pick a later version.